### PR TITLE
Several accessibility improvements

### PR DESCRIPTION
**Pull request by jakrams**

### CODE REVIEW

File: ArrtSource/App/SettingsDlg.cpp [hunk 1]
```diff
@@ -114,6 +126,24 @@ void SettingsDlg::on_TestStorage_clicked()
 {
     ApplyStorage();
 
+    if (StorageName->text().isEmpty())
+    {
+        QMessageBox::warning(this, "Test storage account connection", "Please provide a valid storage name.", QMessageBox::Ok);
```
Comment (MichaelZp0) [inline ArrtSource/App/SettingsDlg.cpp:131]: I would change it to 'Please provide a valid storage account name'.

File: ArrtSource/App/SettingsDlg.cpp [hunk 2]
```diff
@@ -114,6 +126,24 @@ void SettingsDlg::on_TestStorage_clicked()
...
+        return;
+    }
+
+    if (StorageKey->text().isEmpty())
+    {
+        QMessageBox::warning(this, "Test storage account connection", "Please provide a valid storage key.", QMessageBox::Ok);
```
Comment (MichaelZp0) [inline ArrtSource/App/SettingsDlg.cpp:137]: Again I would specify that it is the storage account.

File: ArrtSource/Rendering/UI/StartSessionDlg.ui [hunk 3]
```diff
@@ -210,7 +213,7 @@
      <item row="1" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
-        <string>Max Time (H:MM):</string>
+        <string>Max Session Duration (H:MM):</string>
```
Comment (MichaelZp0) [inline ArrtSource/Rendering/UI/StartSessionDlg.ui:216]: Will this be narrated correctly? You wrote 'H colon MM' above, but now it is 'H:MM'. Both look fine to me, but I would like it to be consistent.

**Review by MichaelZp0 — CHANGES_REQUESTED** [review]

File: ArrtSource/Rendering/UI/StartSessionDlg.ui (hunk 3 above)
Comment (jakrams) [reply ArrtSource/Rendering/UI/StartSessionDlg.ui:216]: This is the label, it won't be narrated at all. The edit box right next to it is the one that gets focus and is narrated, that's why I used the 'colon' there.

**Review by MichaelZp0 — APPROVED** [review]

**Pull request merged**